### ISSUE TITLE
Pa-400 series on pan os 11 & later accept 100K Ip now

### ISSUE THREAD

**Issue by nrgneilo** [romainmarcoux/malicious-ip#7]
Please update or create new URL for 100K ip for Pa-400 series.  Pa-400 series on pan os 11& later accept 100K Ip and 1M Url

https://docs.paloaltonetworks.com/pan-os/11-1/pan-os-admin/policy/use-an-external-dynamic-list-in-policy/external-dynamic-list

Thanks.

**Comment by nrgneilo** [comment]
Good catch! I'm mistaken.  Yeah, restricted to 50K ip.  Sorry.  i thought the URL section was IP's   That's to bad.

**Comment by romainmarcoux** [comment]
Yes, it's a shame. That's why I'm keeping the full-40k file.
FortiGates have much higher limits.

**Comment by nrgneilo** [comment]
From my understanding,  the FW will only use up to 50k Ip's say in a auto ip block at one time. 

  "When the maximum supported IP address limit is reached on the firewall, the firewall generates a syslog message. The IP addresses in predefined IP address lists do not count toward the limit."



Sounds like you can still have the entire list?    I will confirm with PA.

**Comment by romainmarcoux** [comment]
I agree that the wording of the sentence is not explicit.
You can test on a PAN-OS FW by adding the full-aa and full-ab segments (131k IP each) and see if it is 50k IP per segment or 50k IP in total that are taken into account.
